### DISCUSSION TITLE
Empty state in Charts

### DISCUSSION THREAD

**Discussion by ataldev** [mantinedev/mantine#6369] [Q&A]
Hi there, 
What is the best way to show an empty state when there is no data for the charts?

**Comment by rtivital** [comment]
```tsx
if (data.length === 0) return <div>EmptyState</div>
return <AreaChart />
```

**Reply by ataldev** [reply]
Thanks. 
It would be very nice to have smart charts. So if the data passed to it is empty, the chart renders but displays a message saying no data available. And maybe a blurred background. 
Something like:
![image (1).png](https://github.com/mantinedev/mantine/assets/62022057/867544fc-ce10-4595-a0b8-c0c23b8ed791)
